pom.xml

```diff
@@ -214,6 +214,27 @@
       </resource>
     </resources>
     <plugins>
+      <!-- Do not deploy the root POM -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-deploy-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>default-deploy</id>
+            <phase>deploy</phase>
+            <goals>
+              <goal>deploy</goal>
+            </goals>
+            <configuration>
+              <skip>true</skip>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
```

Comment:
Do we need to have flatten plugin?

Reply:
Every module's pom has a reference to the parent: 
![image](https://user-images.githubusercontent.com/6621793/204660842-f14c3b3c-99d1-426f-a1e3-5178d8a2e831.png)

If we're not going to deploy the parent, then we'll need to flatten the pom so that it doesn't reference the parent.

Reply:
Why are we trying to avoid deploying the parent?

Reply:
We were trying to keep the deployment similar to gradle. The gradle deployment doesn't deploy a parent jar so we've added in the flatten plugin and skipped deploying the parent pom

Reply:
I see.